specification/_types/analysis/analyzers.ts

```diff
@@ -56,7 +56,7 @@ export class FingerprintAnalyzer {
    *
    * @server_default _none_
    */
-  stopwords?: StopWords
+  stopwords?: StopWordLanguage | string[]
```

Comment:
This will cause inconveniences for users in statically typed languages.

Could we please continue using the `StopWords` type here?

`StopWords` is currently defined as:

```ts
export type StopWords = string | string[]
```

Changing that to:

```ts
export type StopWords = StopWordLanguage | string[]
```

while keeping this file "as is", should do the trick.

What do you think?

Reply:
@flobernd I think I got what you mean. Made a change in db8c13039816597ff87429c8aafe25f25384e890. Let me know if that looks better to you.

Reply:
Perfect now! Thank you @JoshMock